docs/modules/operation/pages/deep-dive/admin/jmx-config-generator/cli.adoc

```diff
@@ -54,9 +51,9 @@ $ cd /path/to/opennms/bin
 $ ./jmx-config-generator
 ----
 
-TIP: When invoked without parameters, the tool's usage and help information is printed.
+TIP: When invoked without parameters, the tool's usage and help information is displayed.
 
-The JMX Config Generator uses sub-commands for different configuration generation tasks.
+The JMX Configuration Generator uses sub-commands for different configuration generation tasks.
```

Comment:
```suggestion
The JMX Configuration Generator uses subcommands for different configuration generation tasks.
```
Ugh. I don't like the way that looks, but MS style guide says not to hyphenate words beginning with sub- except in certain circumstances.